# HG changeset patch
# User Alexander Wilms <f.alexander.wilms@gmail.com>
# Date 1547136069 -3600
#      Thu Jan 10 17:01:09 2019 +0100
# Branch stable
# Node ID 2d7a7cf5bfb70445e1b2a0ab265fd3970b035577
# Parent  41efc737ceb915f1b49b20aff5ac49274988e7ec
org.octave.Octave.appdata.xml: Add German and French description (bug #55279).

diff --git a/etc/icons/org.octave.Octave.appdata.xml b/etc/icons/org.octave.Octave.appdata.xml
--- a/etc/icons/org.octave.Octave.appdata.xml
+++ b/etc/icons/org.octave.Octave.appdata.xml
@@ -39,25 +39,66 @@
       numerical computations. It provides capabilities for the numerical
       solution of linear and nonlinear problems, and for performing other
       numerical experiments. It also provides extensive graphics capabilities
       for data visualization and manipulation. Octave is normally used through
       its interactive command line interface, but it can also be used to write
       non-interactive programs. The Octave language is quite similar to
       Matlab so that most programs are easily portable.
     </p>
+    <p xml:lang="de">
+      GNU Octave ist eine höhere, interpretierte Programmiersprache, die
+      hauptsächlich für numerische Berechnungen gedacht ist. Sie bietet
+      Funktionen für die numerische Lösung linearer und nichtlinearer Probleme
+      und für die Durchführung anderer numerischer Experimente. Darüber hinaus
+      bietet es umfangreiche Grafikfunktionen zur Datenvisualisierung und
+      -manipulation. Octave wird normalerweise mittels einer interaktiven
+      Befehlszeile verwendet, kann aber auch zum Schreiben nicht interaktiver
+      Programme verwendet werden. Die Octave-Programmiersprache ist sehr
+      ähnlich zu Matlab, so dass die meisten Programme leicht übertragbar sind.
+    </p>
+    <p xml:lang="fr">
+      GNU Octave est un langage de programmation interprété, destiné
+      principalement au calcul numérique. Il fournit des fonctionnalités
+      pour la résolution de problèmes linéaires et non-linéaires, et pour toutes
+      sortes d'expériences numériques.  Il s'accompagne d'un système complet
+      de visualisation et de manipulation des données. Octave s’utilise
+      généralement via son environnement en lignes de commandes interactif,
+      mais il peut aussi être utilisé pour écrire des programmes non
+      interactifs. Le langage Octave est très similaire à Matlab de telle sorte
+      que les programmes Matlab sont très faciles à porter sous Octave.
+    </p>
     <p>
       Octave has extensive tools for solving common numerical linear algebra
       problems, finding the roots of nonlinear equations, integrating ordinary
       functions, manipulating polynomials, and integrating ordinary
       differential and differential-algebraic equations. It is easily
       extensible and customizable via user-defined functions written in
       Octave's own language, or using dynamically loaded modules written in
       C++, C, Fortran, or other languages.
     </p>
+    <p xml:lang="de">
+      Octave verfügt über umfangreiche Werkzeuge zur Lösung gängiger
+      numerischer linearer Probleme der Algebra, zum Auffinden der Nullstellen
+      nichtlinearer Gleichungen, zur Integration gewöhnlicher Funktionen, zur
+      Manipulation von Polynomen und zur Integration gewöhnlicher
+      Differentialgleichungen und differential-algebraischer Gleichungen. Es
+      ist leicht erweiter- und anpassbar mittels benutzerdefinierter
+      Funktionen, die in Octaves eigener Programmiersprache geschrieben wurden,
+      oder über dynamisch geladene, in C++, C, Fortran oder anderen Sprachen
+      geschriebene Module.
+    </p>
+    <p xml:lang="fr">
+      Octave dispose de nombreux outils pour résoudre les problèmes classiques
+      d'algèbre linéaire, trouver les racines d'équations non-linéaires,
+      intégrer les fonctions ordinaires et les équations différentielles.
+      Il peut très facilement être étendu en définissant de nouvelles
+      fonctions écrites en langage Octave ou en utilisant des modules
+      chargés dynamiquement, écrits en C++, C, Fortran ou autres langages.
+    </p>
   </description>
 
   <screenshots>
     <!--
       From: https://www.freedesktop.org/software/appstream/docs/
       All screenshots should have a 16:9 aspect ratio, and should have
       a width that is no smaller than 620px. Screenshots should be in PNG
       or JPEG format. PNG is the preferred format.
